builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -268,6 +270,14 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 							Optional: true,
 							ForceNew: true,
 						},
+
+						"bootable_vmdk_path": &schema.Schema{
+							// TODO: Add ValidateFunc to check if path exists
```

Comment:
I figure its a 'good to have' but not a blocker for my first stab.  There aren't checks for things like network label / folder / datacenter / ect, which we could do, but just aren't at the yet.  I figure its a low hanging fruit to call out.